app/jobs/purge_job.rb

```diff
@@ -5,7 +5,14 @@ class PurgeJob < ApplicationJob
   queue_as :default
 
   def perform(*_args)
+    account_sid = ENV['TWILLIO_API_ACCOUNT']
+    auth_token = ENV['TWILLIO_API_KEY']
     Patient.purge_eligible.find_each(batch_size: 5000) do |monitoree|
+      # Delete Twilio data
+      client = Twilio::REST::Client.new(account_sid, auth_token)
+      messages = client.messages.list(to: Phonelib.parse(patient.primary_telephone, 'US').full_e164)
```

Comment:
if some members of a household are purged but others are not, and they share the same phone number for reporting, would this be a problem in that it would delete the Twilio data for the members of the household that aren't supposed to be purged yet?

Reply:
Hmm, that is a possible edge case (though, if it did happen, I think it would actually be okay). Will give it some thought, but maybe it makes sense to do a quick check for the phone number existing before a delete (so only do a delete if this particular monitoree is the very last one to exist with that exact number). Alternative might be to get fancier with the initial search.